src/www/ui/api/Controllers/UploadController.php

```diff
@@ -891,4 +891,48 @@ public function getMainLicenses($request, $response, $args)
     }
     return $response->withJson($licenses, 200);
   }
+
+  /***
+   * Remove the main license from the upload
+   *
+   * @param ServerRequestInterface $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function removeMainLicense($request, $response, $args)
+  {
+    $uploadId = intval($args['id']);
+    $shortName = $args['shortName'];
+    $returnVal = null;
+    $licenseDao = $this->container->get('dao.license');
+    $clearingDao = $this->container->get('dao.clearing');
+    try {
```

Comment:
Can the scope for this `try/catch` be limited to `removeMainLicense`?

Alternatively, the slim framework already has a exception handler. Thus this block can be ignored.

Reply:
Great, done